### PR TITLE
chore: convert quicksort challenge into lab

### LINKED ISSUE #465: chore: add lab for quick sort in frontend cert(JS only)
There will be lectures videos to talk about algorithms, big O and introduce them to quick sort.
But we need a lab for them to do it themselves.

In the coding interview prep section, we already have this quick sort interactive challenge
https://www.freecodecamp.org/learn/coding-interview-prep/algorithms/implement-quick-sort

Convert that challenge, into the lab format with more user stories and tests

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/js-projects` directory
- [ ] add your completed project in an `script.js` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #465

### CODE REVIEW

File: frontend-cert/js-projects/quicksort/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+1. You should define a function named `quickSort` to implement the quicksort algorithm.
+
+1. The `quickSort` function  should take an array of integers as input and return an array of these integers in sorted order from least to greatest.
```
Comment (zairahira) [inline frontend-cert/js-projects/quicksort/user-stories.md:3]:
```suggestion
1. The `quickSort` function should take an array of integers as input and return an array of these integers in sorted order from least to greatest.
```

File: frontend-cert/js-projects/quicksort/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+1. To implement the algorithm, you should:
+    - choose a pivot value among the elements of the input array (use the first or the last element of the array).
+    - partition the input array into three subarrays made by elements greater than, lesser than, and equal to the pivot value.
+    - recursively call `quickSort` to sort the subarrays.
```
Comment (zairahira) [inline frontend-cert/js-projects/quicksort/user-stories.md:8]:
```suggestion
1. To implement the algorithm, you should:
    - Choose a pivot value among the elements of the input array (use the first or the last element of the array).
    - Partition the input array into three subarrays made by elements greater than, lesser than, and equal to the pivot value.
    - Recursively call `quickSort` to sort the subarrays.
```

**Review by zairahira — COMMENTED** [review]
This lab is looking good, there are no more suggestions from my end.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**